install/updateVersion.sh

```diff
@@ -56,7 +56,6 @@ function gen_file() {
 
 if [[ ! -z ${ISTIO_RELEASE:-} ]]; then
     for target in istio-demo.yaml istio-demo-auth.yaml; do
-        echo XXXXXXXXXXXXXXX $target
```

Comment:
I caught this in the original review, although thought you wanted it in for some reason :)

Reply:
Debug statement. The generation of the release needs cleanup...